tests/scripts/thread-cert/Cert_7_1_05_BorderRouterAsRouter.py

```diff
@@ -183,6 +184,7 @@ def verify(self, pv):
         _pkt = pkts.filter_wpan_src64(MED).\
             filter_wpan_dst64(ROUTER).\
             filter_mle_cmd(MLE_CHILD_UPDATE_REQUEST).\
+            filter(lambda p: len(p.mle.tlv.addr_reg_iid) == 4).\
```

Comment:
let the check more flexible?
```suggestion
            filter(lambda p: len(p.mle.tlv.addr_reg_iid) >= 4).\
```